deny.toml

```diff
@@ -72,7 +72,13 @@ skip-tree = [
 unknown-registry = "deny"
 unknown-git = "deny"
 allow-registry = ["https://github.com/rust-lang/crates.io-index"]
+<<<<<<< HEAD
+allow-git = [
+  "https://github.com/RustCrypto/hashes.git",
+]
+=======
 allow-git = []
+>>>>>>> 109d9f734804a8b9dc0531c0b576e2a0f55a40de
```

Comment:
leftover merge artifacts

Reply:
Rebased to `dl-precompiles`. Now there is no merge commit